src/auth/index.js

```diff
@@ -264,6 +271,10 @@ export default class Auth {
     });
   }
 
+  getLoggedInProviderType() {
+    return this.storage.get(authCommon.USER_LOGGED_IN_PT_KEY);
```

Comment:
[minor] Given that this directly proxies a storage async method, we should probably be better about the resolved output regardless of failure conditions. Under normal circumstances, this should return the type as a string, otherwise, an empty string could suffice. Another option is to throw an exception if `isAuthenticated` resolves to false.

Reply:
An exception could make sense, otherwise I think an `undefined` or `null` is an accurate representation of there not being a provider type.

Reply:
Yeah so if we make it an exception, we will have to be very careful using it with Promise.all by not using it all and instead chaining it with `isAuthenticated` and then `getLoggedInProviderType`. `undefined` or `null` is okay too.